src/resolvers-cassandra/Settings/queries.js

```diff
@@ -6,6 +6,9 @@ const cassandraConnector = require('../../clients/cassandra/CassandraConnector')
 const withRunTime = require('../shared').withRunTime;
 const trackEvent = require('../../clients/appinsights/AppInsightsClient').trackEvent;
 
+const CONNECTOR_TWITTER = 'Twitter'
```

Comment:
Shouldn't these be lowercase like the stream names? [Example](https://github.com/CatalystCode/project-fortis-spark/blob/5c324a98eceabedf58b84dcadaa7b13c712830e9/src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala#L178).